packages/react-adminer/src/components/Edit.tsx

```diff
@@ -48,6 +51,7 @@ const EditChild: React.FC<Props> = ({ entityConfig, entityName, id }) => {
 	const [isSaving, setSaving] = useState(false);
 	const [errorNullable, setErrorNullable] = useState<Record<string, boolean>>({});
 	const isNewForm = id === NEW_KEY;
+	const { Panel } = Collapse;
```

Comment:
Tohle hoď mimo komponentu, jen se "rozbaluje" knihovna, takže to nemusíš dělat při každém renderu.